notebooks/spark/AerospikeSparkScala.ipynb

```diff
@@ -6,35 +6,81 @@
    "source": [
     "# Aerospike Spark Connector Tutorial for Scala\n",
     "\n",
-    "## Tested with Spark connector 3.0.1, Java 8, Apache Spark 3.0.0, Python 3.7  and Scala 2.12.11 and Spylon ( https://pypi.org/project/spylon-kernel/)\n"
+    "## Tested with Spark connector 2.8.0, Java 8, Apache Spark 2.4.0, Python 3.7  and Scala 2.11.12 and Spylon ( https://pypi.org/project/spylon-kernel/)\n"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
Comment this out and mention that it's not required for connector versions 2.8 and 3.1 and later.
conf.set("aerospike.keyPath", "/etc/aerospike/features.conf")